.swcrc.build.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Note that this file is intentionally called `.swcrc.build.json`, so that if
+  // we use SWC for other things in the future, this file will not affect them.
+  "$schema": "https://json.schemastore.org/swcrc",
+  "jsc": {
+    "parser": {
+      "syntax": "typescript"
+    },
+    "target": "es2020"
+  },
+  "sourceMaps": true,
+  "exclude": [
+    ".*__fixtures__.*",
```

Comment:
`tsconfig.build.json` only excludes test files; it doesn't exclude files related to tests. Should we sync `tsconfig.build.json` with this list? Or does it matter?

Reply:
Good catch, we should sync these. There's no point in generating types for something that we don't have JS files for. 😅